### ISSUE TITLE
JumpToDependency: The indent .sass style doesn't grab complete paths

### ISSUE THREAD

**Issue by mrjoelkemp** [mrjoelkemp/Dependents#150]
From https://github.com/mrjoelkemp/Dependents/issues/139#issuecomment-117309701:

---

I am messing around with a new webapp template that uses sass "indent"
syntax (as opposed to scss) and Dependents fails

That being with no quotes and no following ;. If I put them in then no
problem, without and it can't parse it correctly.

@import base/topography

Looks like it found the file name but missed the leading path base/

```
find_base_path: Closest folder with a deprc:
/mnt/AllData/hacking/testing/gulp2-starter/app/assets/stylesheets
find_base_path: Closest folder with a deprc:
/mnt/AllData/hacking/testing/gulp2-starter/app/assets/stylesheets
project_settings: Using found .deprc within:
/mnt/AllData/hacking/testing/gulp2-starter/app/assets/stylesheets/.deprc
project_settings: Using found .deprc within:
/mnt/AllData/hacking/testing/gulp2-starter/app/assets/stylesheets/.deprc
project_settings: setting sass_root for backCompat ./
project_settings: Using these settings: {'node_path': '', 'sass_root':
'./', 'styles_root': './', 'root': '', 'config': '', 'exclude': [],
'build_config': ''}
command_setup: Relative sass_root set to
/mnt/AllData/hacking/testing/gulp2-starter/app/assets/stylesheets
JumpToDependency: region (77, 87)
JumpToDependency: line (64, 87)
JumpToDependency: strings on line []
JumpToDependency: No strings found using word region: typography
JumpToDependency: Extracted Path {'path': 'typography'}
JumpToDependency: Extension found in dependency name
JumpToDependency: Sass lookup for: typography.sass
Executing: node
/home/david/.config/sublime-text-3/Packages/Dependents/node_modules/sass-lookup/bin/cli.js
--filename=/mnt/AllData/hacking/testing/gulp2-starter/app/assets/stylesheets/global.sass
--directory=./ typography.sass
BaseThread: Opening: /opt/sublime_text/typography.sass
Missing file {'_filename': 'BaseThread.py', 'filename':
'/opt/sublime_text/typography.sass'}
error: Dependents
```

**Comment by mrjoelkemp** [comment]
Support was added in 5fe249e4bdce441d2dc5ea52b286b74a606c4dac, but that doesn't seem to handle slashed paths – only single words.

**Comment by mrjoelkemp** [comment]
cc @dkebler. Fixed in version 3.0.1.
